arangod/Aql/ExecutionBlockImpl.cpp

```diff
@@ -1786,19 +1775,27 @@ template <class Executor>
 std::tuple<ExecutionState, SkipResult, SharedAqlItemBlockPtr>
 ExecutionBlockImpl<Executor>::executeWithoutTrace(AqlCallStack stack) {
   if constexpr (isNewStyleExecutor<Executor>) {
-    if (!stack.isRelevant()) {
-      LOG_QUERY("bf029", DEBUG) << "subquery bypassing executor " << printBlockInfo();
-      // We are bypassing subqueries.
-      // This executor is not allowed to perform actions
-      // However we need to maintain the upstream state.
-      SkipResult skippedLocal;
-      typename Fetcher::DataRange bypassedRange{ExecutorState::HASMORE};
-      std::tie(_upstreamState, skippedLocal, bypassedRange) =
-          executeFetcher(stack, _upstreamRequest);
-      return {_upstreamState, skippedLocal, bypassedRange.getBlock()};
+    AqlCallList clientCallList = stack.popCall();
+    AqlCall clientCall;
+    if constexpr (std::is_same_v<Executor, SubqueryEndExecutor>) {
+      // In subqeryEndExecutor we actually manage two calls.
+      // The clientClient is defined of what will go into the Executor.
```

Comment:
```suggestion
      // The clientCall defines what will go into the Executor.
```

(but *which* executor? the current one is the SubqueryEnd one, so the SubqueryEnd executor?